curator/run_cron.py

```diff
@@ -90,13 +102,30 @@ def run_all_jobs(joblist):
 # run jobs now
 run_all_jobs(my_cron)
 
-thehour = int(os.environ.get('CURATOR_CRON_HOUR', 0))
-theminute = int(os.environ.get('CURATOR_CRON_MINUTE', 0))
+thehour = decoded.get('.defaults', {}).get('runhour', None)
+if not thehour:
+    thehour = os.getenv('CURATOR_RUN_HOUR', None)
+if not thehour:
+    thehour = defaults.get('runhour', 0)
+
+theminute = decoded.get('.defaults', {}).get('runminute', None)
+if not theminute:
+    theminute = os.getenv('CURATOR_RUN_MINUTE', None)
+if not theminute:
```

Comment:
I did have it as a python one-liner originally, but based on the feedback from @sosiouxme I split it up this way for readability.